client/cmd/dexc-desktop/config.go

```diff
@@ -0,0 +1,54 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+
+	"decred.org/dcrdex/client/app"
+)
+
+// Config is the configuration for the DEX client application.
+type Config struct {
+	app.Config
+	Kill      bool   `long:"kill" description:"Send a kill signal to a running instance and exit"`
+	LogStdout bool   `long:"stdout" description:"Log to stdout (in addition to the log file)"`
+	Webview   string `long:"webview" description:"Opens a webview window pointing to the provided URL. Does nothing else. Precludes applicability of any other settings."`
+}
+
+func configure() (*Config, error) {
+	// Pre-parse the command line options for one of a few CLI-only settings;
+	// --appdata, --config, --help, or --version.
+	cliCfg := Config{
+		Config: app.DefaultConfig,
+	}
+	if err := app.ParseCLIConfig(&cliCfg); err != nil {
+		return nil, err
+	}
+
+	// Show the version and exit if the version flag was specified.
+	if cliCfg.ShowVer {
+		fmt.Printf("Desktop app version %s (Go version %s %s/%s)\n",
+			app.Version, runtime.Version(), runtime.GOOS, runtime.GOARCH)
+		os.Exit(0)
```

Comment:
Refactor left the `os.Exit` in the helper.

Reply:
I'm not following.

Reply:
I guess `-V` is an ok exception, but this is the reason for the `mainCore` business because only `main` should be doing an `os.Exit`.  Everything else generally should pass an error back up for main to handle.